cmake/README.md

```diff
@@ -8,7 +8,7 @@ The following variables are provided for oneDPL configuration:
 
 | Variable                     | Type   | Description                                                                                   | Default value |
 |------------------------------|--------|-----------------------------------------------------------------------------------------------|---------------|
-| ONEDPL_BACKEND               | STRING | Threading backend; supported values: tbb, dpcpp, dpcpp_only, serial, ...; the default value is defined by compiler: dpcpp for DPC++ and tbb for others | tbb/dpcpp |
+| ONEDPL_BACKEND               | STRING | Threading backend; supported values: tbb, dpcpp, dpcpp_only, tbb, omp, serial, ...; the default value is defined by compiler: dpcpp for DPC++ and tbb for others. These values may conflict with explicit set variables like `-DCMAKE_CXX_FLAGS="-DONEDPL_USE_DPCPP_BACKEND=0"` etc. | tbb/dpcpp |
```

Comment:
```suggestion
| ONEDPL_BACKEND               | STRING | Threading backend; supported values: `dpcpp`, `dpcpp_only`, `tbb`, `omp`, `serial`. The default value is defined by the compiler: `dpcpp` for DPC++ and `tbb` for others. Avoid defining backend control macros (`ONEDPL_USE_TBB_BACKEND`, `ONEDPL_USE_OPENMP_BACKEND`, `ONEDPL_USE_DPCPP_BACKEND`) directly, as they may conflict with the `ONEDPL_BACKEND` setting. | tbb/dpcpp |
```

Reply:
The suggestion above also replaces
> the default value is defined by compiler: dpcpp for DPC++ and tbb for others

with

> The default value is `dpcpp` if SYCL support can be detected and `tbb` otherwise.

It aligns the note with the relevant behaviour.

Reply:
> Avoid defining backend control macros (`ONEDPL_USE_TBB_BACKEND`, `ONEDPL_USE_OPENMP_BACKEND`, `ONEDPL_USE_DPCPP_BACKEND`) directly

Would not it be better to describe what takes precedence in this case? Or is it not deterministic?